### PR TITLE
Add Bengali language for react docs

### PR DESCRIPTION
**Pull request by moshfiqrony**
I have been working as a software engineer for last 6 years. I have started using React since my first project in my organization.

I have added two more maintainer and both have 3+ years of working experience in React. One of them is currently a Lecturer at IUBAT - International University of Business Agriculture and Technology. 

We are based at Bangladesh and we want to create a Bengali version of React docs for the better understanding of the Bengali native language people.

We also have @nomandhoni-cs @shakib609 @nutboltu in the group. They worked on the legacy doc translation for Bengali. 

Maintainers: [ @moshfiqrony, @ahmmedabir9, @nazir021, @nomandhoni-cs, @shakib609, @ahnafyy, @nafistiham ]

### CODE REVIEW

**Comment by moshfiqrony** [comment]
Thank you so much @nomandhoni-cs for your interest. Can you share your previous PR link here?

**Comment by nomandhoni-cs** [comment]
@moshfiqrony After making some progress, React Moved their docs from React.org to React.dev

I am pretty much involved with Open Source, as you can see from my GitHub profile. 

Merged:
https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pull/215
https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pull/216

Not Merged:
https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pull/214
https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pull/213
https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pull/205

**Comment by moshfiqrony** [comment]
I really appreciate the effort. Thanks for sharing the links. I will add you.

**Comment by moshfiqrony** [comment]
@nomandhoni-cs Just wanted to make sure you already saw this [Maintainer responsibilities](https://github.com/reactjs/translations.react.dev/blob/main/maintainer-guide.md#maintainer-responsibilities). If not I would encourage you to have a look at this and let me know you are ok with it.

**Comment by nomandhoni-cs** [comment]
Maintainer responsibilities are almost same as previous react.org docs Maintainer responsibilities. So, I am confortable with the responsibilities

**Comment by nomandhoni-cs** [comment]
Thanks, hope this PR gets merged!

**Comment by shakib609** [comment]
Hello @moshfiqrony 

Continuing from #277

Great that you took the initiative to add Bengali translations for the new website. me and @nutboltu have been mostly maintaining the previous react translations. Would love to start translating the new versions as well.

Can you kindly add me as well to the maintainers list?

You can check my previous translation PRs [here](https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pulls?q=is%3Aclosed+is%3Apr+author%3Ashakib609+).

Cheers. Looking forward to starting the new translations.

**Comment by ahnafyy** [comment]
Came here from [link](https://github.com/reactjs/translations.react.dev/pull/277#issuecomment-1523883856) @moshfiqrony please add me as well as contributor!

p.s. @shakib609 would also like to be added

File: langs/bn.json
Comment (ahnafyy) [inline langs/bn.json:2]: Should it be 'Bengali'?

File: langs/bn.json
Comment (ahnafyy) [inline langs/bn.json:3]: bn or ben?

File: langs/bn.json
Comment (moshfiqrony) [reply langs/bn.json:2]: Thanks for the correction.

**Comment by nutboltu** [comment]
@moshfiqrony Thanks for taking the initiative for the new translation.

I would be happy to be a part of the new journey. However, I am sharing my learnings from my previous experience:

1. It would be good to maintain a [Glossary](https://github.com/stale-reactjs-org-translations/bn.reactjs.org/wiki/Glossary) from the beginning.
2. We can set some rules for Bengali translations only. e.g
    - There shouldn't be any mixture of English and Bengali languages while translating a sentence.
    - Technical keywords shouldn't be translated.
    - A significant page can be breakdown into multiple PRs. It will help the volunteers to translate and keep updated with the changes.
3. A Slack/Discord channel for the maintainers will significantly help in further discussions.
4. Some PRs from the root repositories will require translations and fixing merge conflicts. Setting up a weekly reminder for the maintainers would help address those PRs. Otherwise, the entire translation would need to be updated.

**Comment by ahnafyy** [comment]
> @moshfiqrony Thanks for taking the initiative for the new translation.
> 
> I would be happy to be a part of the new journey. However, I am sharing my learnings from my previous experience:
> 
> 1. It would be good to maintain a [Glossary](https://github.com/stale-reactjs-org-translations/bn.reactjs.org/wiki/Glossary) from the beginning.
> 2. We can set some rules for Bengali translations only. e.g
>    
>    * There shouldn't be any mixture of English and Bengali languages while translating a sentence.
>    * Technical keywords shouldn't be translated.
>    * A significant page can be breakdown into multiple PRs. It will help the volunteers to translate and keep updated with the changes.
> 3. A Slack/Discord channel for the maintainers will significantly help in further discussions.
> 4. Some PRs from the root repositories will require translations and fixing merge conflicts. Setting up a weekly reminder for the maintainers would help address those PRs. Otherwise, the entire translation would need to be updated.

1. Glossary sounds like a fantastic idea! Good call, man. Lets do it 🕺 
2. No mixture alright alright.. I see your point but can one write a sentence in bengali but keep the technical word still in english? 
3. All in for slack/discord! Lets make it.
4. Lets setup a weekend day? Maybe around bangladesh night time and US morning time? 😄

**Comment by nutboltu** [comment]
@ahnafyy 
> No mixture alright alright.. I see your point but can one write a sentence in bengali but keep the technical word still in English?
IMO, we can keep the Technical keywords unchanged. (e.g.: React, Javascript, HTML, CSS). But this decision should come from all the maintainers

**Comment by moshfiqrony** [comment]
@nutboltu that makes total sense. Let's continue it once the pr gets merged and the repo gets created. I have plan for Discord server for communication.

@tesseralis @gaearon Do we need to provide any more information to get this started?

**Comment by nafistiham** [comment]
@moshfiqrony, I would like to join as a maintainer. [I tried contributing to react bn 4 years back](https://github.com/stale-reactjs-org-translations/bn.reactjs.org/pull/46) Farhad Yasir vai did approve my works but unfortunately, I was a novice in using git back then and the branches got mixed up. Now I'd like to start anew.

I have more than 4 years of professional experience. Currently, I am working as a senior engineer at bKash. I have had more than 2 years of experience using React in my last workplace.  I'd like to add that I have authored a [book on mathematics](https://www.amazon.com/Gonit-Olympiad-101-somossa-somadhan/dp/B08CV5MWV9/ref=sr_1_1?qid=1682590950&refinements=p_27%3ANafis+Tiham&s=books&sr=1-1), half of which is rigorous translated work. 

I have read and understood the responsibilities. I aim to adhere to them to the best of my capability.

**Comment by moshfiqrony** [comment]
@nutboltu @shakib609 @nomandhoni-cs  @ahnafyy If you guys think we should add @nafistiham as a maintainer please react with  🚀

**Comment by ahnafyy** [comment]
My suggestions (i.e. opinions)

- We would like people who are familiar with actually working with react/ javascript/ know how the DOM works and have 3-4 years of experience working with it.

Reason: If not, you'll often be confused when the doc will talk about something fairly technical. If you don't truly understand what it's saying clearly you won't be able to translate it a way that remains true to its essence.

- We would like people who are actively working on it!

Reason: If not, what's the point of you being a maintainer? I am also suggesting if there are people who contribute a lot, we think about swapping them with a different person who is a maintainer and is a little inactive.

- We would like people who speak bengali and have experience translating (or skilled enough to do it!)

Reason: If you dont know how to do it haha wait what? how are you doing this? dont you dare say chatGPT or google translate lmao 🤡 

Last thought,
I say lets keep 4-5 maintainers max at first 👥 
We can swap people in and out like have a rotation if someone is being like I am going to be inactive for a bit or is wanting to move on to give someone else a chance to contribute more (or are inactive?) If you're not a maintainer when this PR gets merged that don't mean you won't be able to contribute (or even be one in the future) 😄

**Comment by ahnafyy** [comment]
@gaearon here's some virtual 🍪 🍪 🍪  cookie(s) to lure you in here and get this ball rollin' 😎 🤞

🎾.......🏃🏾..🏃🏾... 💨💨

**Comment by moshfiqrony** [comment]
Please join this community server https://discord.gg/ER3DzHEh

**Review by gaearon — CHANGES_REQUESTED** [review]
As a part of fixing the permission sync process, we've changed how we track languages — please see https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9. 
Could you please update this PR:

- Rebase on top of latest `main`, and add the language to `lang/langs.json` instead
- Include the list of maintainers into the PR comment body instead

Thank you, and sorry for the churn.

**Comment by moshfiqrony** [comment]
Thanks for the feedback @gaearon. Noted what you suggested and will update the PR accordingly.

**Comment by moshfiqrony** [comment]
PR updated as described

**Comment by moshfiqrony** [comment]
> As a part of fixing the permission sync process, we've changed how we track languages — please see https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9.
Could you please update this PR:
>
> - Rebase on top of latest main, and add the language to lang/langs.json instead
> - Include the list of maintainers into the PR comment body instead
>
>Thank you, and sorry for the churn.

Thanks for the feedback @gaearon. PR Updated based on the suggestion.

**Just confirming**

- I can see langs/langs.json not lang/langs.json what you mentioned

**Comment by gaearon** [comment]
https://github.com/reactjs/bn.react.dev/issues/1

**Comment by moshfiqrony** [comment]
Thank you so much @gaearon

**Pull request merged**
